server.py

```diff
@@ -158,104 +184,130 @@ def setStatus(commandName, status, exist = False):
     else:
         return False
     if commandFromSettings.strip() != '':
-        broadlinkControlIniFile = open(path.join(settings.applicationDir, 'settings.ini'), 'w')    
+        broadlinkControlIniFile = open(path.join(settings.applicationDir, 'settings.ini'), 'w')
         settingsFile.set('Status', commandName, status)
         settingsFile.write(broadlinkControlIniFile)
         broadlinkControlIniFile.close()
         return True
     else:
         return False
 
+
 def getStatus(commandName):
     if settingsFile.has_option('Status', commandName):
         status = settingsFile.get('Status', commandName)
         return status
     else:
         return False
 
-def getTempRM():
-    device = broadlink.rm((RMIPAddress, RMPort), RMMACAddress)
+
+def getTempRM(deviceName=None):
+    if deviceName == None:
```

Comment:
I think need to validate first that device is RM

Reply:
Worse, I had deleted all the A1 stuff a long time ago because I don't have one and the old code wasn't forgiving about that.  I just pushed another change that restores the A1 sensors and tries to start integrating getTemp w/device detection